soap/client.go

```diff
@@ -224,6 +226,7 @@ type Envelope struct {
 	XMLName      xml.Name `xml:"SOAP-ENV:Envelope"`
 	EnvelopeAttr string   `xml:"xmlns:SOAP-ENV,attr"`
 	NSAttr       string   `xml:"xmlns:ns,attr"`
+	URNAttr      string   `xml:"xmlns:urn,attr,omitempty"`
 	TNSAttr      string   `xml:"xmlns:tns,attr"`
```

Comment:
You can save the next fella a trouble (in !97) and add `omitempty` for `TNSAttr` as well

Reply:
I've left comment in #97 . It will not work like that =)

Reply:
We have an envelope coming from 3rd party server that have this attribute missing, and xml parser is chocking on that. Will your fix alleviate that?